### PR TITLE
migrate lightFormat to ts

### PR DESCRIPTION
**Pull request by a-korzun**
fixes https://github.com/date-fns/date-fns-typescriptify/issues/113

### CODE REVIEW

**Review by tan75 — APPROVED** [review]
Thank you!

File: src/lightFormat/index.ts [hunk 1]
```diff
@@ -75,12 +75,13 @@ var unescapedLatinCharacterRegExp = /[a-zA-Z]/
  * var result = lightFormat(new Date(2014, 1, 11), 'yyyy-MM-dd')
```
Comment (tan75) [inline src/lightFormat/index.ts:75]: Could you please change `var` to `const` in the examples as well?

**Review by tan75 — CHANGES_REQUESTED** [review]
Please change `var` to `const` in the examples.

**Comment by a-korzun** [comment]
> Please change `var` to `const` in the examples.

Fixed

**Review by tan75 — APPROVED** [review]
Much appreciated!

**Pull request merged**